app/controllers/static_pages_controller.rb

```diff
@@ -1,17 +1,21 @@
 class StaticPagesController < ApplicationController
   def index
     session[:snap_application_id] = nil
+    session[:medicaid_application_id] = nil
```

Comment:
I wonder if it makes sense to move these session-setters into a `before_action`?

Reply:
great idea! done at 9cefe71